tests/test_tiingo_pandas.py

```diff
@@ -114,6 +114,24 @@ def test_intraday_ticker_price(self):
                                             frequency="30Min")
         self.assertGreater(len(prices), 1)
 
+    @vcr.use_cassette('tests/fixtures/ticker_price_with_volume_column.yaml')
```

Comment:
Confirming my understanding these two tests makes sense because "date" is always provided as a column, so the # of columns is always the length of `len(columns.split(",")) + 1` .

Reply:
You are correct, you always receive the `date` column

Reply:
I've updated the _pandas test with your suggestion

Reply:
Updated docstrings and variable names make sense, thank you